templates/gallery/gallery-map.html

```diff
@@ -0,0 +1,318 @@
+<!doctype html>
```

Comment:
create a folder called 3p inside this template and move this file in there (The only stuff that needs to be there inside gallery is packable AMP HTML)

Reply:
Do you mean to just move the file to `templates/gallery/3p/gallery-map.html`, and other than, continue using as-is? Or do I need to make a modification to this file?

Reply:
Yes